prometheus/config/node_exporter/textfile_collectors/files/smartmon.sh.jinja

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
```

Comment:
This is fine.  For `template-formula` with subcomponents, I'm not sure what is best file structure.  We could have this implementation (all good) or have a `prometheus/config/files/` universal directory for all stuff.  For this PR I'm happy with your logical implementation. OK!

Reply:
The textfile exporters are pretty ad-hoc in comparison to the core structure of this formula. I'd rather separate it.